### ISSUE TITLE
Add a text label to the theme dark/light mode-changing button

### ISSUE THREAD

**Issue by samcunliffe** [UCL-ARC/python-tooling#537]
See parent issue for explanation in James' email.

## Definition of done

The mode-changing button has an alt-text or meaningful label: It doesn't say "moon" (or "sun") when read by a screen reader. 

## Docs

Solution could be an [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label).

**Comment by samcunliffe** [comment]
Huh. Should have been auto-closed. Probably I didn't use the correct word. 
Solved via #543.

**Comment by paddyroddy** [comment]
> Huh. Should have been auto-closed. Probably I didn't use the correct word. Solved via [#543](https://github.com/UCL-ARC/python-tooling/pull/543).

You had Fixes
* #num

I think you have to do Fixes #num
